projects/plugins/jetpack/modules/search/class.jetpack-search-helpers.php

```diff
@@ -719,6 +719,16 @@ public static function get_asset_version( $file ) {
 			: JETPACK__VERSION;
 	}
 
+	/**
+	 * Get alphanumberic Jetpack version
+	 *
+	 * @since 9.8.0
+	 * @param string $replace The string non-alphanumeric charactors are replaced by.
+	 * @return string $script_version Alphanumberic version number.
+	 */
+	public static function get_alphanumeric_version( $replace = '-' ) {
+		return strtolower( preg_replace( '/[^A-Za-z0-9]/', $replace, JETPACK__VERSION ) );
```

Comment:
I think it's worth including an example of what `JETPACK_VERSION` looks like.

Reply:
added in 2252bbc